style.css

```diff
@@ -0,0 +1,278 @@
+:root {
+    --main-blue: #0067fc;
+    --light-blue: #f0f8ff;
+    --dark-blue: #0048cd;
+    --main-white: #fff;
+    --title-size: 22px;
+    --header-height: 65px;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+main {
+    margin-top: var(--header-height);
+    min-height: calc(100vh - 80px - 50px );
+}
+
+header {
+    width: 100%;
+    height: var(--header-height);
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: var(--main-white);
+}
+
+.header-container {
+    max-width: 1200px;
+    display: flex;
+    justify-content: space-between;
+    padding-right: 15px;
+    margin: 0 auto;
+}
+
+header .logo {
+    width: 100px;
+}
+
+.custom-button {
+    border: none;
+    background: none;
+    font-family: monospace;
+    height: 30px;
+    font-size: 12px;
+    color: var(--main-white);
+    background-color: var(--main-blue);
+    border-radius: 6px;
+    transition: 0.3s;
+}
+
+.custom-button:hover {
+    cursor: pointer;
+    color: var(--main-blue);
+    background-color: var(--main-white);
+    border: 1px solid var(--main-blue);
+}
+
+nav {
+    display: flex;
+    align-items: center;
+    gap: 10px;
+}
+
+nav a {
```

Comment:
Using element selector might be a little dangerous because it limits you where you can use it. I would recommend to use a flat selector like `.nav-link`